__tests__/app.test.js

```diff
@@ -1,87 +1,199 @@
-            .expect(400)
-            .then((res) => {
-            expect(res.body.msg).toBe('Invalid id');
-            });
-        })
-        test("STATUS 404 - article doesn't exist", () => {
-            return request(app).get('/api/articles/999').expect(404).then((res) => {
-            expect(res.body.msg).toBe("Article doesn't exist");
-             });
-        })
-    })
-})
\ No newline at end of file
+  describe("STATUS 200", () => {
+    test("responds with status 200 and returns relevant article", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((res) => {
+          const date = new Date(1594329060000).toJSON();
+          expect(res.body.article).toEqual({
+            username: "butter_bridge",
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            body: "I find this existence challenging",
+            created_at: date,
+            votes: 100,
+          });
+        });
+    });
+  });
+  describe("STATUS 400s", () => {
+    test("Status 400 - invalid id", () => {
+      return request(app)
+        .get("/api/articles/invalidId")
+        .expect(400)
+        .then((res) => {
+          expect(res.body.msg).toBe("Invalid input data type");
+        });
+    });
+    test("STATUS 404 - article doesn't exist", () => {
+      return request(app)
+        .get("/api/articles/999")
+        .expect(404)
+        .then((res) => {
+          expect(res.body.msg).toBe("Article not found!");
+        });
+    });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  describe("STATUS 200", () => {
+    test("Responds with 200 and returns relevant article with incremented votes", () => {
+      const newVote = { inc_votes: 1 };
+      const date = new Date(1594329060000).toJSON();
+      return request(app)
+        .patch("/api/articles/1")
+        .send(newVote)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.updatedArticle).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: date,
+            votes: 101,
+          });
+        });
+    });
+    test("Responds with 200 and returns relevant article with decremented votes", () => {
+      const newVote = { inc_votes: -100 };
+      const date = new Date(1594329060000).toJSON();
+      return request(app)
+        .patch("/api/articles/1")
+        .send(newVote)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.updatedArticle).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: date,
```

Comment:
think carefully about which properties are actually relevant to your tests; to what extent is your code DRY here?